packages/zeebe-element-templates-json-schema/test/fixtures/invalid-zeebe-input-type.js

```diff
@@ -20,61 +20,16 @@ export const template = {
         'type': 'zeebe:input',
         'name': 'bar'
       }
```

Comment:
The fixture name is now misleading. Let's add some absurd type instead, e.g. `type: Car`

Reply:
Missed that one, I adjusted it for every other test. No idea why I did not do it with this one 🙈

adjusted and squashed with previous commits